src/components/AddingAnswer/index.tsx

```diff
@@ -90,7 +94,7 @@ export const AddingAnswer = ({
           onPress={onPressCorrectAnswerHandler}
           isChecked={isChecked}
           disabled={!!disabledCheckbox}
-          fillColor={Color.GreenLight}
+          fillColor={checkingForDuplicate ? Color.Red : Color.GreenLight}
```

Comment:
if your variable return boolean, its name should start from `is, are, has` and etc. https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx#:~:text=2.%20Naming%20Convention%20for%C2%A0Booleans